routes/api/individualsearchresult.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var router = express.Router();
+var Hotel = require('../../models/Hotel');
+var Booking = require('../../models/booking');
+
+router.get('/individual-search-result/:id', (req,res) =>{
```

Comment:
This should be 
`router.get('/:id', (req,res) =>{`

Reply:
Since all the routes in this file are already `/api/individualsearchresult` by default, any thing you define in the routes here simply add on to that, so what you have here is:
`/api/individualsearchresult/individual-search-result/:id`.

Reply:
if we receiving query string we don't need the ':id', just `router.get('/individual-search-result`

so they will pass something like `individual-search-result?id=5c7&checkin=2019-3-7`

Reply:
My suggestion is to use:
`/api/hotel` in **server.js** and
`/:id` in **this file** to get a final path of `localhost:5000/api/hotel/:id`

Reply:
I think the ‘:id’ means dynamic parameters, not quite sure. If it’s dynamic parameters it will mess up other routes, just in case we can use ‘localhost:5000/api/hotel/individual’

Reply:
It is a dynamic parameter, but your earlier suggestion about passing a larger query string in the URL is the right move. So it could be:
`localhost:5000/api/hotel?id=1234&checkin=2019-3-7&checkout=...`
The URL doesn't matter too much since the public facing route will be different, maybe:
`localhost:3000/hotel/1234?checkin=2019-3-7&checkout=...`